src/chapters/setup-intro-data-types/setup.rst

```diff
@@ -1,2 +1,218 @@
+
+For the entirety of this course, we will be coding in Java. We need to not only have
+Java installed on our machines, but also some accompanying technologies to allow
+us to run and edit our Java code.
+
+Java Development Kit
+--------------------
+
+Installing Java means downloading a package of software called the **Java Development Kit**,
+or **JDK**, for short. The JDK contains software the tools needed to develop and
+run Java code, namely the the Java compiler, **javac**, and the 
+**Java Virtual Machine (JVM)**. 
+
+While the compiler is responsible for processing Java code into machine readable
+code, the JVM allows us to run that code on any computer. These tools 
+together, downloaded as the JDK, give us the means to write, compile, and run Java
+on our machines.
+
+A step-by-step walk-through of the process:
+
+We write code in Java, the code is passed through the compiler program, which 
+translates Java into **bytecode**, a language readable by the JVM. In the JVM,
+bytecode is translated to machine code, which is then executed by your computer.
+
+The JVM gives Java more flexibility than other compiled programming languages because
+it will translate bytecode into the appropriate machine code, depending on the 
+operating environment.
+
+
+Install the JDK
+---------------
+
+In order to learn Java, you need to have the JDK installed on your computer. 
+
+Open a terminal window on your machine and enter the following command:
+
+.. sourcecode:: bash
+
+   java -version
+
+If the response returns a version 13 or higher, you can move on to the section below,
+:ref:`cli-java`.
+
+If you do not have a version of Java at 13 or higher, you can download 
+it `here <https://www.oracle.com/technetwork/java/javase/downloads/jdk13-downloads-5672538.html>`__.
+The relevant install link is on the bottom of the page:
+
+.. figure:: figures/installJava.png
+   :alt: Install Java
+
+   Install Java
+
+To install, you must first select *Accept License Agreement*, then select any of 
+the file type options for your operating system. Once you have completed the 
+installation steps, return to your terminal and enter ``java -version`` again to 
+ensure Java has been installed.
+
+.. _cli-java:
+
+CLI Java
+--------
+
+Let's write a simple "Hello, World" program and watch the JDK in action. 
+
+In the future, we'll be doing most of our Java coding with the IntelliJ IDE. 
+IntelliJ contains many features to help us write Java properly and easily, 
+including its own compiler. For now though, we'll use a simpler text editor 
+so we can demonstrate what we get with the JDK.
+
+In the text editor of your choice, create and save a file called 
+``HelloWorld.java`` and include the code below:
+
+.. sourcecode:: java
+   :linenos:
+
+   public class HelloWorld {
+
+      public static void main(String[] args) {
+
+         System.out.println("Hello, World");
+      }
+
+   }
+
+We'll discuss the syntax of this program soon, but you can likely trust your gut
```

Comment:
The description of ``System.out.println`` should not take that many words, so I'd recommend explaining it here.  We can defer discussion about the two ``public`` lines for now and focus on what line 5 accomplishes.

Adding this detail here saves us from having to embed it with other topics later in the chapter.